### ISSUE TITLE
Singular.jl@master display `1*x1` instead of the usual behavior `x1`

### ISSUE THREAD

**Issue by mohamed-barakat** [oscar-system/Singular.jl#322]
```
julia> using Singular
Singular.jl, based on
                     SINGULAR                                 /
 A Computer Algebra System for Polynomial Computations       /  Singular.jl: 0.4.2-DEV-dev
                                                           0<   Singular   : 0.0.2-4
 by: W. Decker, G.-M. Greuel, G. Pfister, H. Schoenemann     \
FB Mathematik der Universitaet, D-67653 Kaiserslautern        \


julia> R,(x1,x2,x3) = Singular.PolynomialRing(Singular.QQ,[ "x1", "x2", "x3" ]);

julia> x1
1*x1

julia>
```

**Comment by thofma** [comment]
Will be fixed by #323 (after we have figured out some other things).

**Comment by mohamed-barakat** [comment]
Thank you very much. Then you are free to close this issue earlier.
